dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/BaseClasses/BaseGenerator.mo

```diff
@@ -40,6 +40,8 @@ partial model BaseGenerator "Base dynamic model for generators based on SignalN
   parameter Boolean limUQUp0 "Whether the maximum reactive power limits are reached or not (from generator voltage regulator), start value";
   parameter Types.ActivePowerPu PRef0Pu "Start value of the active power set point in pu (base SnRef) (receptor convention)";
   parameter QStatus qStatus0 "Start voltage regulation status: Standard, AbsorptionMax, GenerationMax";
+  parameter Real UDeadBand(min = 0) "voltage deadband around the target";
+  parameter Real QDeadBand(min = 0) "ractive power deadband around the target";
```

Comment:
I would suggest defining them in BasePV.mo instead since GeneratorPQ also inherits from BasGenerator.mo

Reply:
UDeadBand is not used in GeneratorPQ this is why I put it here

Reply:
Yes exactly, I think UDeadBandPu and QDeadBandPu should be defined in BasePV.mo and not in BaseGenerator.mo,  and QDeadBandPu should be defined as you did in BasePQProp.mo.
Because at it is defined now, UDeadBandPu is also a parameter for GeneratorPQ

Reply:
Done